constellation-app/constellation
# Delimited file import filter

### Description of the Change
The Filter in the preview table of Delimited File Importer is broken. It doesn't load any data in the graph for any input in the filter.
Rows that match the filter criteria should be imported into the graph.


### Alternate Designs
N/A


### Why Should This Be In Core?
Better UX


### Benefits
Better UX


### Possible Drawbacks
N/A


### Verification Process

1. Create a file containing the following, and call it tst.csv
a,b,c,d
1,2,3,4
5,6,7,8

2. Select File->Import->From Delimited File...
3. Open the above csv file using "+" button
4. Add the `Identifier` `Source Node Attribute` on to the column `a`
5. Select Destination `New Analytic Graph`
6. Enter `Row !=1` in the Filter -> Row 1 (5,6,7,8) should be Greyed out
Now press the Import button -> There should be a node `1` in the graph.
7. Enter `Row !=0` in the Filter -> Row 0 (1,2,3,4) should be Greyed out.
Now press the Import button ->There should be a node `5` in the graph.


-->

### Applicable Issues
#603
